### PR TITLE
Add function to iterate over ingress annotations to include whitelist…

### PR DESCRIPTION
**Pull request by Vierlip**
…-ranges

### CODE REVIEW

File: .changeset/seven-bobcats-play.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"comet-admin-v5": major
```
Comment (kaufmo) [inline .changeset/seven-bobcats-play.md:2]: why is this a major change? shouldnt this be a minor?

Comment (Vierlip) [reply .changeset/seven-bobcats-play.md:2]: This was the output from running `npx changeset` for the given Chart. Should I change it to minor for all PRs ?

Comment (Vierlip) [reply .changeset/seven-bobcats-play.md]: Version was changed to minor

File: .changeset/seven-bobcats-play.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"comet-admin-v5": minor
+---
+
+Add function to iterate over ingress annotations to include whitelist-ranges
```
Comment (dkarnutsch) [inline .changeset/seven-bobcats-play.md:5]:
```suggestion
Add possibility to add annotations to ingress (e.g. in use by NGINX Ingress Controller)
```

**Review by dkarnutsch — CHANGES_REQUESTED** [review]

File: .changeset/seven-bobcats-play.md (hunk 2 above)
Comment (Vierlip) [reply .changeset/seven-bobcats-play.md]: Fixed. 

e146b9675909af3614e1a28db00ff5685e0f5187

**Review by dkarnutsch — APPROVED** [review]

**Review by Schreglmann — APPROVED** [review]

**Pull request merged**